### PR TITLE
add embeddings.md to readthedocs

### PR DESCRIPTION
**Pull request by yingyibiao**
### PR types

Function optimization
### PR changes

Docs
### Description

add embeddings.md to readthedocs

### CODE REVIEW

File: paddlenlp/embeddings/README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 |  参数 | 类型  | 属性  |
 | ------------ | ------------ | ------------ |
-| embedding_name | **string**  | 预训练embedding名称，可通过paddlenlp.embeddings.list_embedding_name或[Embedding 模型汇总](../../docs/embeddings.md)查询。 |
+| embedding_name | **string**  | 预训练embedding名称，可通过paddlenlp.embeddings.list_embedding_name或[Embedding 模型汇总](../../docs/model_zoo/embeddings.md)查询。 |
```
Comment (ZeyuChen) [inline paddlenlp/embeddings/README.md]: paddlenlp.embeddings.list_embedding_name() ?

**Review by ZeyuChen — COMMENTED** [review]
函数的化是得用()结尾

File: paddlenlp/embeddings/README.md (hunk 1 above)
Comment (yingyibiao) [reply paddlenlp/embeddings/README.md]: done

**Review by ZeyuChen — APPROVED** [review]
LGTM

**Pull request merged**